sqlx-core/Cargo.toml

```diff
@@ -41,3 +41,6 @@ uuid = { version = "0.8.1", default-features = false, optional = true }
 
 [dev-dependencies]
 matches = "0.1.8"
+
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
This and the one in `sqlx-macros` shouldn't be necessary unless people are pulling up the docs for these specifically instead of `sqlx`.

Reply:
It can't hurt I suppose. You know we could add a redirect from the sqlx-core docs to sqlx. Devious. :see_no_evil: